### PR TITLE
Using BOM

### PR DESCRIPTION
**Pull request by jglick**
@oleg-nenashev FYI another.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -67,91 +74,56 @@
...
-      <version>${workflow-step-api-plugin.version}</version>
     </dependency>
     <dependency>
       <groupId>org.jenkins-ci.plugins</groupId>
       <artifactId>structs</artifactId>
-      <version>1.17</version>
-    </dependency>
-    <dependency>
-      <groupId>org.jenkins-ci.plugins.workflow</groupId>
-      <artifactId>workflow-api</artifactId>
```
Comment (jglick) [inline pom.xml]: Transitive dependencies not explicitly mentioned in sources (more or less) can be found using

```sh
mvn dependency:analyze
```

**Comment by jglick** [comment]
> The `bom` groupId is a bit vague

Not sure what you mean by this; could you explain?

**Comment by jvz** [comment]
`io.jenkins.tools.bom`

What is this?

**Comment by jglick** [comment]
Oh, sorry, did not get what you were asking. https://github.com/jenkinsci/bom#introduction

**Pull request merged**